CMakeLists.txt

```diff
@@ -50,5 +50,5 @@ target_link_libraries(llvm_bc_parsing_example PUBLIC LLVMBitReader LLVMBitWriter
 
 target_compile_options(llvm_bc_parsing_example PUBLIC
   $<$<C_COMPILER_ID:GNU>:-Wall -Wextra -Werror -std=gnu89>
-  $<$<C_COMPILER_ID:Clang>:-Wall -Wextra -Weverything -Werror -std=gnu89>
+  $<$<C_COMPILER_ID:Clang>:-Wall -Wextra -Werror -std=gnu89>
```

Comment:
`-Weverything` complains because `switch` does not contain every possible `case` (even though it includes a `default`).